### PR TITLE
doc: update readme

### PR DESCRIPTION
**Pull request by kerzhner**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# Fast Exit
+# Stingy Asymmetric Fast Exit protocol (FAST)
```
Comment (lalexgap) [inline readme.md:1]: Should FAST be SAFE?

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 
-This is a prototype of a fast exit protocol that allows for fast and cheap exits from L2 to L1.
+This is a prototype of the FAST protocol. The protocol allows for secure, trustless fund transfers between chains. The protocol is particularly cost efficient when the gas fees for one chain are meaningfully different (cheaper or more expensive) than another chain. For example, SAFE is useful for transferring tokens from an optimistic rollup to Ethereum mainnet without waiting for the rollup finalization delay.
+
+The protocol is described in detail in the [SAFE paper](https://www.notion.so/statechannels/Paper-Submission-WIP-L2-Asymmetric-Fast-Exits-cf0b29e8656d4c3e8edfe8329b2fa67e).
```
Comment (lalexgap) [inline readme.md:5]: Renamed the document so the link would be cleaner: https://statechannels.notion.site/SAFE-Protocol-cf0b29e8656d4c3e8edfe8329b2fa67e

File: readme.md (hunk 1 above)
Comment (kerzhner) [reply readme.md:1]: Good catch. SAFE and FAST :)

**Review by lalexgap — APPROVED** [review]
👍 

Would it be worth adding a small blurb that says `yarn test` can be used to run some (rudimentary) tests?

**Comment by kerzhner** [comment]
> Would it be worth adding a small blurb that says `yarn test` can be used to run some (rudimentary) tests?

I think that's a good call.

**Pull request merged**